public-html/404.php

```diff
@@ -0,0 +1,13 @@
+<?php
+http_response_code(404);
+require $_SERVER['DOCUMENT_ROOT'] . '/../vendor/autoload.php';
+
+use template\Page;
+
+$new = new Page;
+$content = <<<EOT
+<div class="main_content">
+<h1 style="text-align:center; color:red"> 404 Page Not Found! </h1>
```

Comment:
remove inline css and you correct indentation

Reply:
@rahulsavaria  done